vassal-app/src/main/java/VASSAL/build/module/Map.java

```diff
@@ -2622,15 +2622,28 @@ public void windowClosing(WindowEvent e) {
           if (splitPane.isBottomVisible()) {
             final Dimension d = g.getPlayerWindow().getSize();
             final GlobalPrefs p = (GlobalPrefs) Prefs.getGlobalPrefs();
+            final PlayerWindow window = g.getPlayerWindow();
+            final Rectangle screen = SwingUtils.getScreenBounds(window);
+            final int w = window.getWidth();
+            final int h;
             if (Boolean.TRUE.equals(p.getOption(MAIN_WINDOW_REMEMBER).getValue())) {
               p.setDisableAutoWrite(true);
               p.getOption(MAIN_WINDOW_HEIGHT).setValue((int)d.getHeight());
               p.getOption(MAIN_WINDOW_WIDTH).setValue((int)d.getWidth());
               p.saveGlobal();
               p.setDisableAutoWrite(false);
+
+              h = (int)d.getHeight();
```

Comment:
d.height will be an int, you can use that and avoid the cast.

Reply:
It was pooching me down with "double" issues for some reason. Or you're saying just directly access the field w/o the getter/setter? So that's an int?

Reply:
Yes, Dimension, Point, Rectangle all have public members you can simply use.

Reply:
For some (insert appropriate invective here) reason, in those classes, the width and height are int's, but the getters return doubles.